ibis/bigquery/tests/test_client.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+import pytest
+import pandas as pd
+import pandas.util.testing as tm
+
+import ibis
+import ibis.util
+import ibis.expr.types as ir
+from ibis.bigquery import client as ibc
+from ibis.bigquery.tests import conftest as conftest
+
+
+pytestmark = pytest.mark.bigquery
+
+
+def test_fixture_state(client_with_table, df):
+    bq_dataset = client_with_table._proxy.get_dataset(
+        client_with_table.dataset_id)
+    bq_table = client_with_table._proxy.get_table(
+        conftest.TABLE_ID, client_with_table.dataset_id)
+    assert bq_dataset.exists()
+    assert bq_table.exists()
+
+    df_tuples = list(tuple(v.tolist()) for (k, v) in df.iterrows())
+    table_tuples = list(bq_table.fetch_data())
```

Comment:
This test might be a bit flaky due to the use of `table.insert_data()`, which uses the streaming buffer. Rows in the streaming buffer don't appear in the list of rows from a table right away. (In fact, it could be as much as [90 minutes to get out of the streaming buffer](https://cloud.google.com/bigquery/streaming-data-into-bigquery#dataavailability).)